### DISCUSSION TITLE
Cant get the Plex URL to work...

### DISCUSSION THREAD

**Discussion by MrFauxHawk** [meisnate12/Plex-Meta-Manager#2007] [Q&A] [answered]
Using docker compose on a unbuntu server edition.  using the ip found on the private IP section of plex remote access but im getting this error:

| Traceback (most recent call last):                                                                 |
|   File "/lsiopy/lib/python3.11/site-packages/urllib3/connection.py", line 198, in _new_conn        |
|     sock = connection.create_connection(                                                           |
|            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^                                                           |
|   File "/lsiopy/lib/python3.11/site-packages/urllib3/util/connection.py", line 85, in create_connection |
|     raise err                                                                                      |
|   File "/lsiopy/lib/python3.11/site-packages/urllib3/util/connection.py", line 73, in create_connection |
|     sock.connect(sa)                                                                               |
| TimeoutError: timed out                                                                            |

**Comment by chazlarson** [comment]
Kometa inside the container can't see IPs on the host unless you have the container attached to the host network.

**Reply by MrFauxHawk** [reply]
Here is my Kometa compose file:

#Plex Meta Manager - Media Library 
  kometa:
    image: lscr.io/linuxserver/kometa:latest
    container_name: kometa
    security_opt:
      - no-new-privileges:true
    restart: unless-stopped
    networks:
      - default
    environment:
      PUID: $PUID
      PGID: $PGID
      TZ: $TZ      
      KOMETA_RUN: "true"
      KOMETA_OVERLAYS_ONLY: "true"
      KOMETA_RUN_LIBRARIES: "Movies"
    volumes:
      - $DOCKERDIR/appdata/kometa/config:/config

**Reply by chazlarson** [reply]
That container is on the docker default network.  Where is Plex?

**Reply by MrFauxHawk** [reply]
Same default network

**Reply by chazlarson** [reply]
And what are you using as the Plex URL?

**Reply by chazlarson** [reply]
It should be an internal docker IP, or you could put them both on a named docker network and use the container name

**Reply by MrFauxHawk** [reply]
172.18.0.15 : 32400 which i grabbed from looking at my plex private IP

**Reply by chazlarson** [reply]
OK, sure, that seems fine.  You're using the unofficial lsio container; maybe that has something to do with it.

You are specifying the plex URL as `http://172.18.0.15:32400`, yeah?

**Reply by chazlarson** [reply]
Kometa's just attempting to open a connection and that attempt is timing out.  You could try increasing the timeout.  Is Plex perhaps busy doing maintenance or something?

**Reply by MrFauxHawk** [reply, marked as answer]
Got it all to work, thanks for your help, plex wasnt on the default docker network as i thought it was :)
